nixos/modules/services/web-servers/nginx/default.nix

```diff
@@ -1163,6 +1173,14 @@ in
         '';
       }
 
+      {
+        assertion = all (host: !(host.proxyPass && host.uwsgiPass != null)) (attrValues virtualHosts);
+        message = ''
+          Options services.nginx.service.virtualHosts.<name>.proxyPass and
+          services.nginx.virtualHosts.<name>.uwsgiPass are mutually exclusive.
+        '';
+      }
```

Comment:
This assertion is causing the eval failure. This should recurse through the locations of a virtual host instead because those options are part of locations.